puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/foremanCentosJob.groovy

```diff
@@ -25,7 +26,8 @@ pipeline {
                 runPlaybook(
                     playbook: 'playbooks/setup_forklift.yml',
                     inventory: cico_inventory('./'),
-                    options: ['-b']
+                    options: ['-b'],
+                    extraVars: ['forklift_install_from_galaxy': install_from_galaxy]
```

Comment:
Doesn't this mean that we now set this variable for every forklift job? I think it would be cleaner to determine extraVars either as an empty array or hardcoded set to true if `params.type == 'pulpcore'`.

Reply:
every forklift setup job. but yes. While I see no harm in setting it to `false` for every job but pulpcore, I can also go and refactor that into setting extravars based on type